Snippets/Mouse/mouseClicked.sublime-snippet

```diff
@@ -0,0 +1,9 @@
+<snippet>
+    <content><![CDATA[
+    	mouseClicked(${1:event}){
```

Comment:
Missing return type, and see the other snippets for formatting (need a space between closing parens and opening brace.